hassio/dashboard/hassio-addons.html

```diff
@@ -1,39 +1,40 @@
-<link rel="import" href="../../bower_components/polymer/polymer-element.html">
-<link rel="import" href="../../bower_components/paper-card/paper-card.html">
-<link rel="import" href="../../bower_components/paper-item/paper-item.html">
-<link rel="import" href="../../bower_components/paper-item/paper-item-body.html">
+<link rel='import' href='../../bower_components/polymer/polymer-element.html'>
+<link rel='import' href='../../bower_components/paper-card/paper-card.html'>
 
 <link rel='import' href='../../src/util/hass-mixins.html'>
+<link rel='import' href='../../src/components/hassio-card-content.html'>
+<link rel='import' href='../../src/resources/hassio-style.html'>
 
-<dom-module id="hassio-addons">
+<dom-module id='hassio-addons'>
```

Comment:
Just a heads up, your quote preference is making code review more difficult as it's no longer clear what things are changes and which ones are style tweaks.

Reply:
Ok, I will avoid this for other PRs